typescript/sdk/src/middleware/account/InterchainAccount.ts

```diff
@@ -56,4 +75,161 @@ export class InterchainAccount extends RouterApp<InterchainAccountFactories> {
     );
     return new InterchainAccount(helper.contractsMap, helper.multiProvider);
   }
+
+  async deployAccount(
+    chain: ChainName,
+    config: AccountConfig,
+  ): Promise<Address> {
+    const originDomain = this.multiProvider.tryGetDomainId(config.origin);
+    if (!originDomain) {
+      throw new Error(
+        `Origin chain (${config.origin}) metadata needed for deploying ICAs ...`,
+      );
+    }
+    const localRouter = this.router(this.contractsMap[chain]);
+    await this.multiProvider.handleTx(
+      chain,
+      localRouter[
+        'getDeployedInterchainAccount(uint32,address,address,address)'
+      ](
+        originDomain,
+        config.owner,
+        bytes32ToAddress(await localRouter.routers(originDomain)),
+        bytes32ToAddress(await localRouter.isms(originDomain)),
+      ),
+    );
+    const account = await localRouter[
+      'getLocalInterchainAccount(uint32,address,address,address)'
+    ](
+      originDomain,
+      config.owner,
+      bytes32ToAddress(await localRouter.routers(originDomain)),
+      bytes32ToAddress(await localRouter.isms(originDomain)),
+    );
+    if (
+      (await this.multiProvider.getProvider(chain).getCode(account)) === '0x'
+    ) {
+      throw new Error('Interchain account deployment failed');
+    }
+    return account;
+  }
+
+  // meant for ICA governance to return the populatedTx
+  async getCallRemote(
+    chain: ChainName,
+    destination: ChainName,
+    innerCalls: CallData[],
+  ): Promise<CallData> {
+    const localRouter = this.router(this.contractsMap[chain]);
+    const quote = await localRouter.quoteGasPayment(
+      this.multiProvider.getDomainId(destination),
+    );
+    const icaCall: CallData = {
```

Comment:
any reason not to use `localRouter.populateTx.callRemote`?

Reply:
done

Reply:
I dont understand wdym
`CallData` and `PopulatedTransaction` share `to`, `data`, and `value`